### PR TITLE
Add API with GET health endpoint

### PR DESCRIPTION
**Pull request by shermosa**

### CODE REVIEW

File: beekeeper-api/pom.xml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-maven-plugin</artifactId>
+        <version>${springframework.boot.version}</version>
+        <configuration>
+          <requiresUnpack>
+            <!-- We are hitting an issue introduced in hive-common-2.1.1 and fixed in https://issues.apache.org/jira/browse/HIVE-17155
```
Comment (patduin) [inline beekeeper-api/pom.xml:33]: if hive-common is not a dependency in this project you can remove this.

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,7 +3,9 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
-## [3.1.1] - TBD
+## [4.0.0] - TBD
```
Comment (javsanbel2) [inline CHANGELOG.md:6]: If we are going to deploy today, let's put today's date

File: beekeeper-api/README.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+## Beekeeper API
+
+In progress :) !
```
Comment (javsanbel2) [inline beekeeper-api/README.md:3]: I would put better a short description of what is going to do with Beekeeper API and what you are pushing right now. You can say that it is in progress.

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,7 +3,9 @@ All notable changes to this project will be documented in this file.
...
+### Added
+- Adding `beekeeper-api` module.
```
Comment (javsanbel2) [inline CHANGELOG.md:8]: What about putting Added? I never saw the -ing verb here. (Are you going to put the release date?)

Comment (shermosa) [reply CHANGELOG.md:8]: I put the release date in another PR

**Review by javsanbel2 — APPROVED** [review]

**Pull request merged**